libraries/console_helpers.php

```diff
@@ -12,14 +12,18 @@ function str_to_number($string)
     return (string) filter_var($string, FILTER_SANITIZE_NUMBER_INT);
 }
 
-function choose($message, $availableValues)
+function choice($message, $availableValues, $errorText = null)
 {
     do {
         $command = ask($message);
         $isDataValid = in_array($command, $availableValues);
 
+        if (is_null($errorText)) {
```

Comment:
please move it out of the loop to ake this operation once and not on the each loop iteration